crates/matrix-sdk-crypto/src/types/device_keys.rs

```diff
@@ -190,6 +196,8 @@ impl From<Ed25519PublicKey> for DeviceKey {
     }
 }
 
+/// A de/serialization helper for [`DeviceKeys`] which maps the `keys` to/from
+/// [`DeviceKey`]s.
```

Comment:
very light suggestion: One of the two mentions to `DeviceKey` seems redundant; you get to choose which one :D

Reply:
one is a mention of `DeviceKey`, the other `DeviceKeys`. They are quire different!